src/view.js

```diff
@@ -187,24 +203,10 @@ Marionette.View = Marionette.AbstractView.extend({
   // Internal method to initialize the regions that have been defined in a
   // `regions` attribute on this layoutView.
   _initializeRegions: function(options) {
-    var regions;
     this._initRegionManager();
 
-    regions = Marionette._getValue(this.regions, this, [options]) || {};
-
-    // Enable users to define `regions` as instance options.
-    var regionOptions = this.getOption.call(options, 'regions');
-
-    // enable region options to be a function
-    regionOptions = Marionette._getValue(regionOptions, this, [options]);
-
-    _.extend(regions, regionOptions);
-
-    // Normalize region selectors hash to allow
-    // a user to use the @ui. syntax.
-    regions = this.normalizeUIValues(regions, ['selector', 'el']);
-
-    this.addRegions(regions);
+    this._addRegions(this.regions, [options]);
+    this._addRegions(this.getOption.call(options, 'regions'), [options]);
```

Comment:
Because the regions can both be supplied directly in the `regions` attribute or in the `options`?